src/components/App/App.js

```diff
@@ -5,16 +5,25 @@ import React, { Component } from 'react';
 import classNames from 'classnames';
 import MentorsList from '../MentorsList/MentorsList';
 import Filter from '../Filter/Filter';
+import Content from '../Content/Content';
 import Logo from '../Logo';
 import SocialLinks from '../SocialLinks/SocialLinks';
+import Modal from '../Modal/Modal';
 import shuffle from 'lodash/shuffle';
 import { toggle, get } from '../../favoriteManager';
 
+
+
 // const serverEndpoint = 'http://localhost:3001';
 class App extends Component {
   state = {
     mentors: shuffle(mentors),
     favorites: get(),
+    modal: {
+      title: null,
+      content: null,
+      onClose: null
```

Comment:
Is it needed? Especially in `state`?

```suggestion

```

Reply:
yeah the reason I set this is to make `Modal Component` reach-full, imagine u need to get modal Close Callback when you request for a modal open `openModal('My Title', <MyComponent />, ()=>{ /* if modal closed, then do this*/ });`

Reply:
I understand but I think it shouldn't be on the `state`. Anyway, I'll merge this and maybe we will think how to improve the code later.